src/components/settings/SettingsSSOTab/steps/NewSSOConfigConfirmStep.tsx

```diff
@@ -22,13 +21,13 @@ const IncognitoPopover = () => (
       </Popover>
       )}
   >
-    <Hyperlink>
+    <Button variant="link" size="inline">
```

Comment:
[inform] Resolves a TypeScript, since `destination` is required. Given it seems this component intends to show text "incognito window" as a link to indicate there's a tooltip when clicked, changing it to be a `Button` styled as an inline link is more appropriate than `Hyperlink`.